EIPS/eip-2035.md

```diff
@@ -0,0 +1,75 @@
+operations by 69k gas. However, in light of proposal in EIP-2028 (link will be provided when EIP is merged), the increase can be made much smaller.
+
+## Specification
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
+Not very formal at the moment, but will be formalised with more research and prototyping. Gas of operations `SLOAD` and `SSTORE` increases by `X` gas
+for those operations that access storage slots that were not previously accessed during the same transaction.
+
+Future variant (will be possible after the implentation of the *Gross contract size acccounting*) is researched, where the increase is varied
+depending on the size of the contract storage, i.e. `SLOAD` and `SSTORE` for smaller contracts will be cheaper.
+
+## Rationale
+<!--The rationale fleshes out the specification by describing what motivated the design and why particular design decisions were made. It should describe alternate designs that were considered and related work, e.g. how the feature is supported in other languages. The rationale may also provide evidence of consensus within the community, and should discuss important objections or concerns raised during discussion.-->
+[EIP-1884](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1884.md) seeks to increase the gas cost of `SLOAD` but using a different justification
+(latency of the execution as described in the Motivation). This EIP is likely to increase the cost of `SLOAD` by a larger amount, therefore partially
+(because EIP-1884 also proposed other increases) supersedes EIP-1884.
+
+EIP-2028 (link will be provided when EIP is merged) describes the model that can be used for deciding the gas cost of data transmission. It is relevant
+because in the stateless client regime `SSTORE` and `SLOAD` operations add more data to be transmitted (as well as computation to verify the proofs).
+
+The main alternate design is the rent proportional to the size of the contract storage, which unfortunately introduces a serious griefing
+vulnerability problem, and so far the solution seems to be in redesigning and rewriting smart contracts in a way, which makes them not vulnerable.
+However, this approach is likely to be very expensive on the non-technical (ecosystem) level.
+
+## Backwards Compatibility
+<!--All EIPs that introduce backwards incompatibilities must include a section describing these incompatibilities and their severity. The EIP must explain how the author proposes to deal with these incompatibilities. EIP submissions without a sufficient backwards compatibility treatise may be rejected outright.-->
+This change is not backwards compatible and requires hard fork to be activated.
```

Comment:
This section supposed to explain what adverse effects this change can have on already deployed contracts.

Reply:
I have added some initial description

Reply:
Does this sentence needs to be kept in? It doesn't seem to be useful.

Reply:
There are some changes that do not require hard fork, but still are requirements for other changes, so I say the sentence is still useful